examples/test-app/app.js

```diff
@@ -52,12 +54,19 @@ journey(app, {
     Contact,
     CheckYourAnswers,
     ExitNorthernIreland,
-    Exit
+    Done,
+    Error
   ],
   session: {
     redis: { url: config.redisUrl },
     cookie: { secure: false }
-  }
+  },
+  apiUrl: `${baseUrl}/api/submit`
+});
+
+app.post('/api/submit', bodyParser.json(), (req, res) => {
+  const response = { status: 'ok', originalBody: req.body };
```

Comment:
dont see much point in declaring it

Reply:
True, my original idea had been to send a different value back if you gave certain answers, to demonstrate the action failing. Will do that now

Reply:
ok, I've put in a check that fails the action if the two last names don't match. That will atleast demo the functionality.